### DISCUSSION TITLE
„Brainstorming Matter“?

### DISCUSSION THREAD

**Discussion by Graefer** [Supergiovane/node-red-contrib-knx-ultimate#357] [Ideas]
I just read that you are considering to implement Matter and thus to make KNX devices ready to work with everything which supports Matter. That’s a fantastic idea! I once considered to suggest to make a bridge to HomeKit. But I thought this was too much work and unrealistic. And now you consider a Matter Bridge yourself, which is even better!
However, I would be very interested. Have you considered an optional paid license for this?
Good luck!

**Comment by Supergiovane** [comment]
Hi Graefer!
No, there will be no paid licenses ever. ;-)
